plugins/module_utils/network/sonic/facts/logging/logging.py

```diff
@@ -117,4 +117,15 @@ def get_logging_configuration(self):
 
         logging_config['remote_servers'] = logging_servers
 
+        """Get the syslog security profile configurations in the device"""
+        request = [{"path": "data/openconfig-system:system/openconfig-system-ext:syslog/config", "method": GET}]
+        try:
+            response = edit_config(self._module, to_request(self._module, request))
+        except ConnectionError as exc:
+            self._module.fail_json(msg=str(exc), code=exc.code)
```

Comment:
Please handle GET failure in facts for unsupported SONiC version.
```suggestion
        except ConnectionError as exc:
            if 'Resource not found' in str(exc):
                return logging_config
            self._module.fail_json(msg=str(exc), code=exc.code)
```

Reply:
Addressed